cmd/node/config/config.toml

```diff
@@ -654,6 +654,7 @@
    HeartbeatRefreshIntervalInSec        = 60
    HideInactiveValidatorIntervalInSec   = 3600
    DurationToConsiderUnresponsiveInSec  = 60
+   HeartbeatDisableEpoch                = 650
```

Comment:
please move it to `enableEpocs.toml` - also it should be set to a much lower epoch in node's configuration. This value (650) would be set accordingly in elrond config mainnet

Reply:
fixed, PR on elrond-config-mainnet will follow